components/style/properties/helpers.mako.rs

```diff
@@ -185,6 +185,7 @@
         if property is None:
             return ""
     %>
+    /// ${property.spec}
```

Comment:
Maybe worth to be a bit more descriptive (using also `${property.name}`.

Ideally we could come up with pretty good messages, but that's not required to land this.